src/modules/Crowbar.rb

```diff
@@ -195,7 +202,15 @@ def Read
       @conduit_map = network["conduit_map"] || []
 
       @template_crowbar = json2hash(@crowbar_file)
-      @users = @template_crowbar["attributes"]["crowbar"]["users"] rescue {}
+
+      begin
+        crowbarrc = IniFile.load("/etc/crowbarrc")
+        @admin_user = crowbarrc["default"]["username"]
```

Comment:
This might crash when `crowbarrc["default"]` is missing (is `nil`).

Reply:
@lslezak no, the inifile gem doesn't work that way: `crowbarrc` is not a hash but another object, and `crowbarrc[foo]` will never be `nil` -- in the worst case, it'll be `{}`